src/ros_license_toolkit/package.py

```diff
@@ -121,7 +129,7 @@ def _run_scan_and_save_results(self):
         for (root, _, files) in os.walk(self.abspath):
             files_rel_to_pkg = [self._get_path_relative_to_pkg(
                 os.path.join(root, f)) for f in files]
-            for pattern in IGNORED:
```

Comment:
What happened to these? I think we should still also ignore everything in `IGNORED` by default. And the config file  should be per package, right?